lib/lotus/validations/attribute_definer.rb

```diff
@@ -330,9 +328,16 @@ def define_writer(name)
 
         # @since 0.2.2
         # @api private
-        def define_reader(name)
+        def define_reader(name, options = {})
           define_method(name) do
-            @attributes.get(name)
+            attribute = @attributes.get(name)
+            if attribute.nil? && !options[:type].nil? && !options[:default].nil?
```

Comment:
I don't like a lot this implementation (the code), we have 3 cases and we support three cases, what do you think guys?
```
attribute :name, type: String, presence: true, default: 1
```
```
attribute :level, default: 1
```
```
attribute :level
```